app/services/crown_court_outcome_creator.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+class CrownCourtOutcomeCreator < ApplicationService
+  GUILTY_VERDICTS = %w[GUILTY_BUT_OF_LESSER_OFFENCE_BY_JURY_CONVICTED
+                       GUILTY_CONVICTED
+                       GUILTY_BY_JURY_CONVICTED
+                       GUILTY_BUT_OF_LESSER_OFFENCE_CONVICTED].freeze
+
+  def initialize(defendant:)
+    @defendant = defendant
+  end
+
+  def call
+    return unless result_is_a_conclusion?
+
+    {
+      ccooOutcome: trial_outcome,
+      caseEndDate: case_end_date,
+      appealType: nil
+    }
+  end
+
+  private
+
+  def result_is_a_conclusion?
+    defendant&.dig(:offences)&.any? { |offence| offence.dig(:verdict).present? }
+  end
+
+  def trial_outcome
+    return 'CONVICTED' if defendant&.dig(:offences)&.all? do |offence|
+      GUILTY_VERDICTS.include? offence.dig(:verdict, :verdictType, :categoryType)
+    end
+    return 'PART CONVICTED' if defendant&.dig(:offences)&.any? do |offence|
+      GUILTY_VERDICTS.include? offence.dig(:verdict, :verdictType, :categoryType)
+    end
+
+    'AQUITTED'
```

Comment:
Not sure if this was intentional for compatibility or just a typo.
```suggestion
    'ACQUITTED'
```

Reply:
Unfortunately that's how it's spelled in the MAAT database, so we need to keep it like that for compatibility...

![image](https://user-images.githubusercontent.com/28729201/84121196-b92a5c00-aa2e-11ea-9158-0bb74679e18c.png)

I've added a comment explaining this though, so that it doesn't get accidentally 'fixed' in future.